tests/main/apt-hooks/task.yaml

```diff
@@ -13,9 +13,8 @@ debug: |
     
 execute: |
     echo "Ensure we have a snap catalog in our cache"
-    while ! test -s /var/cache/snapd/commands.db; do
```

Comment:
I think we need to keep a test that checks if we do the catalog update on startup.

Reply:
that could be done as a unit test (i think we don't have it right now)